### DISCUSSION TITLE
Documentation Wrong for dispatchAtom example

### DISCUSSION THREAD

**Discussion by benson00077** [pmndrs/jotai#2281] [Bug report]
The [documentation](https://jotai.org/docs/guides/composing-atoms#action-atoms) for the `dispatchAtom` example appears to consistently throw an error for me:

```js
// continued from the previous code
export const dispatchAtom = atom(null, (_get, set, action) => {
  if (action === 'INC') {
    set(incAtom)
  }
  if (action === 'DEC') {
    set(decAtom)
  }
  throw new Error('unknown action')
}
```
Maybe `return` got missed in each if statement?

Thanks!

**Comment by dai-shi** [comment]
Looks like so. Can you open a PR?

**Reply by benson00077** [reply]
Sure! 
I just created PR here https://github.com/pmndrs/jotai/pull/2284

**Reply by AjaxSolutions** [reply]
Can you just use "if else else" to avoid having two return statements?

```
export const dispatchAtom = atom(null, (get, set, action) => {
  if (action === 'INC') {
    set(incAtom);
  } else if (action === 'DEC') {
    set(decAtom);
  } else {
    throw new Error('unknown action');
  }
});
```

**Reply by dai-shi** [reply]
Yeah, I like it.

**Reply by benson00077** [reply]
Looks good! I just amended the commit in the PR.
